librz/bin/bobj.c

```diff
@@ -771,15 +771,15 @@ static RzList *get_sections_or_segment(RzBinObject *obj, bool is_segment) {
 /**
  * \brief Get list of \p RzBinSection representing only the sections of the binary object.
  */
-RZ_API RzList *rz_bin_object_get_sections(RzBinObject *obj) {
+RZ_API RZ_OWN RzList *rz_bin_object_get_sections(RzBinObject *obj) {
 	rz_return_val_if_fail(obj, NULL);
 	return get_sections_or_segment(obj, false);
 }
 
 /**
  * \brief Get list of \p RzBinSection representing only the segments of the binary object.
  */
-RZ_API RzList *rz_bin_object_get_segments(RzBinObject *obj) {
+RZ_API RZ_OWN RzList *rz_bin_object_get_segments(RzBinObject *obj) {
```

Comment:
```suggestion
RZ_API RZ_OWN RzList *rz_bin_object_get_segments(RZ_NONNULL RzBinObject *obj) {
```

Same also for the others above